apps/forms/base.py

```diff
@@ -87,20 +149,20 @@ def validate_subdomain(self, subdomain_input):
             if Subdomain.objects.filter(subdomain=subdomain_input).exists():
                 error_message = "Wow, you just won the lottery. Contact us for a free chocolate bar."
                 raise forms.ValidationError(error_message)
-            return subdomain
+            return subdomain, False
```

Comment:
Could you please add a docstring explaining what this boolean values suppose to mean?

Reply:
And I'd like to suggest defining a custom named tuple for this in `types_`:

```py
class SubdomainTuple(NamedTuple):
    subdomain: str
    is_created_by_user: bool

    def __str__(self) -> str:
        return subdomain
```

Because it's a tuple, you can still unpack it, but you can also just call `str(sub_tup)` if needed